ledger-core/virtual/integration/method_test.go

```diff
@@ -1423,18 +1412,12 @@ func Test_MethodCall_HappyPath(t *testing.T) {
 		}
 
 		commontestutils.WaitSignalsTimed(t, 10*time.Second, executeDone)
-		commontestutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
 
-		// increment pulse twice for stop SMPreservedStateReport
-		server.IncrementPulseAndWaitIdle(ctx)
 		server.IncrementPulseAndWaitIdle(ctx)
 
-		commontestutils.WaitSignalsTimed(t, 10*time.Second, stateReportSend)
+		commontestutils.WaitSignalsTimed(t, 10*time.Second, typedChecker.VStateReport.Wait(ctx, 1))
 		commontestutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
 
-		require.Equal(t, 1, typedChecker.VStateReport.Count())
```

Comment:
It's more readable than `Count()` in typedChecker mock

Reply:
return this changes